### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by mordamax**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -207,25 +207,28 @@ once the application starts.
 
 ## Run the application <a name="development-run"></a>
 
-1. [Set up the GitHub App](#github-app)
-2. [Set up the application](#setup)
+1. [Set up the GitHub App](#github-app).
```
Comment (joao-paulo-parity) [inline README.md]: Remove the `.`

File: README.md [hunk 2]
```diff
@@ -207,25 +207,28 @@ once the application starts.
 
 ## Run the application <a name="development-run"></a>
 
-1. [Set up the GitHub App](#github-app)
-2. [Set up the application](#setup)
+1. [Set up the GitHub App](#github-app). 
+2. [Set up the command-bot application](#setup)
```
Comment (joao-paulo-parity) [inline README.md:211]: Why is this needed? "The application" can only mean command-bot here because this is its README

File: README.md [hunk 3]
```diff
@@ -207,25 +207,28 @@ once the application starts.
...
+    Along with creating app, replace *placeholders* in `.env.cjs` with required data.
+    
     During development it's handy to use a [smee.io](https://smee.io/) proxy,
     through the `WEBHOOK_PROXY_URL` environment variable, for receiving GitHub
     Webhook Events in your local server instance.
 
-3. [Install the GitHub app](#github-app-installation) in a GitHub repository
-4. Set up a GitLab repository in [`$GITLAB_PUSH_NAMESPACE`](./.env.example.cjs)
-  to run the commands for the GitHub repository (Step 3)
+4. [Install the GitHub app](#github-app-installation) in a GitHub repository
```
Comment (joao-paulo-parity) [inline README.md:219]: Jumped from 2 to 4?

File: README.md [hunk 4]
```diff
@@ -207,25 +207,28 @@ once the application starts.
...
+1. [Set up the GitHub App](#github-app). 
+2. [Set up the command-bot application](#setup)
 
+    Along with creating app, replace *placeholders* in `.env.cjs` with required data.
+    
     During development it's handy to use a [smee.io](https://smee.io/) proxy,
     through the `WEBHOOK_PROXY_URL` environment variable, for receiving GitHub
     Webhook Events in your local server instance.
 
-3. [Install the GitHub app](#github-app-installation) in a GitHub repository
```
Comment (joao-paulo-parity) [inline README.md:217]: Why remove this?

File: .env.example.cjs [hunk 5]
```diff
@@ -66,10 +67,9 @@ process.env.GITLAB_ACCESS_TOKEN_USERNAME ??= "placeholder"
 // process.env.PIPELINE_SCRIPTS_REF ??= "placeholder"
 
 /*
-  This private key's file can be generated and downloaded from
+  Download private key file from
   https://github.com/settings/apps/[app-name].
-  If you need to calculate the Base64 of the value manually, that can be done
```
Comment (joao-paulo-parity) [inline .env.example.cjs:71]: The text is saying **If you need**. Quite explicitly it's not needed in this case because the conversion is done by `Buffer.from`. I don't mind this being removed but I don't think it's necessary to remove it.

File: .env.example.cjs [hunk 6]
```diff
@@ -41,14 +41,15 @@ process.env.GITLAB_PUSH_NAMESPACE ??= "placeholder"
...
   The default image to be used on GitLab jobs for bot commands which don't
-  specify a custom image
+  specify a custom image (like `quay.io/buildah/stable`)
 */
 process.env.GITLAB_JOB_IMAGE ??= "placeholder"
 
 /*
   GITLAB_ACCESS_TOKEN token needs the following scopes:
   - "write_repository"
   - "read_api"
```
Comment (joao-paulo-parity) [inline .env.example.cjs:51]: Remove `"read_api"` as its redundant with `"api"`

File: README.md [hunk 7]
```diff
@@ -207,25 +207,28 @@ once the application starts.
 
 ## Run the application <a name="development-run"></a>
 
-1. [Set up the GitHub App](#github-app)
-2. [Set up the application](#setup)
+1. [Set up the GitHub App](#github-app).
+2. [Set up the command-bot application](#setup)
+
+    Along with creating app, replace *placeholders* in `.env.cjs` with required data.
```
Comment (joao-paulo-parity) [inline README.md:213]: This is included in the Setup section, more exactly `setup-environment-variables`

**Review by joao-paulo-parity — CHANGES_REQUESTED** [review]

File: README.md (hunk 7 above)
Comment (mordamax) [reply README.md]: removed
